.mk/development.mk

```diff
@@ -179,12 +179,15 @@ endif
 	@echo -e "\n==> Redeploying..."
 	kubectl rollout status -n $(OPERATOR_NS) --timeout=60s deployment netobserv-controller-manager
 	kubectl wait -n $(OPERATOR_NS) --timeout=60s --for condition=Available=True deployment netobserv-controller-manager
-	kubectl rollout status -n $(NAMESPACE) --timeout=60s deployment netobserv-plugin
-	kubectl wait -n $(NAMESPACE) --timeout=60s --for condition=Available=True deployment netobserv-plugin
+	@echo -e "\n==> Wait a moment before plugin pod is fully redeployed"
 
 .PHONY: set-release-kind-downstream
 set-release-kind-downstream:
+ifeq ("", "$(CSV)")
```

Comment:
This is for kind cluster deploy which doesn't use CSV

Reply:
I think there's a confusion :-) It's not for KIND clusters, it's "release kind" like in "downstream kind of release"